packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -43,11 +46,12 @@ export const generateTestDataForPage = async (
   locale: string,
   projectStructure: ProjectStructure
 ): Promise<any> => {
+  const siteConfigPath =
+    projectStructure.sitesConfigDomain?.getAbsolutePath() ??
+    projectStructure.sitesConfigRoot.getAbsolutePath();
   const siteStreamPath = path.resolve(
     process.cwd(),
-    projectStructure.sitesConfigRoot.getAbsolutePath() +
-      "/" +
-      projectStructure.siteStreamConfig
+    siteConfigPath + "/" + projectStructure.siteStreamConfig
```

Comment:
do we want to use path.join here instead? sometimes other os (windows) are weird